packages/react/src/primitives/Accordion/types.ts

```diff
@@ -0,0 +1,55 @@
+import { ElementType, PrimitiveProps, BaseViewProps } from '../types/view';
+
+export interface BaseAccordionProps extends BaseViewProps {
```

Comment:
While you're here, aren't the Base* props the ones we don't want to export?

Reply:
Yup, but I just export them from this file. They aren't publicly exported.